core/src/main/java/tech/pegasys/web3signer/core/multikey/metadata/SigningMetadata.java

```diff
@@ -41,4 +41,8 @@ protected SigningMetadata(final KeyType keyType) {
   public KeyType getKeyType() {
     return keyType;
   }
+
+  public void validate() throws SigningMetadataException {
```

Comment:
the fact that no one else needs the validate method, feels like its the wrong abstraction

Reply:
addressed. Refactored to a separate class.